client/package.json

```diff
@@ -8,14 +8,15 @@
     "lint": "vue-cli-service lint"
```

Comment:
Remove the changes made to this file and pack-lock.json. Just copy and paste from main